main.go

```diff
@@ -109,11 +109,28 @@ func main() {
 				MaxRows:     config.Transport.QueueStorage.MaxRows,
 				MaxFileAge:  time.Duration(config.Transport.QueueStorage.MaxFileAgeSeconds) * time.Second,
 			},
+			DB: db,
+			// TODO: should producer and consumer use different directories?
```

Comment:
Yes

Reply:
added `produce_data_dir` and `consumer_data_dir` config options